### DISCUSSION TITLE
Problems to UPDATE Z2M

### DISCUSSION THREAD

**Discussion by wwyyrrmm** [Koenkk/zigbee2mqtt#23310] [General]
Hardware Raspberry
OS:Debian GNU/Linux 12 (bookworm)
Node.js: v20.11.1

I tryed to update Z2M
cd /opt/zigbee2mqtt
./update.sh

Output:
Checking Zigbee2MQTT status...
Creating backup of configuration...
Checking out changes to package-lock.json...
Updated 0 paths from the index
Updating...
Already up to date.
Installing dependencies...
npm WARN deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm WARN deprecated @humanwhocodes/config-array@0.11.14: Use @eslint/config-array instead
npm WARN deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm WARN deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm WARN deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead

added 799 packages, and audited 800 packages in 6m

87 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
Building...

> zigbee2mqtt@1.39.0 build
> tsc && node index.js writehash

why are the "packages" DEPRECATED?
what is the solution?

thx for hints - Carsten

**Comment by wwyyrrmm** [comment]
Z2M facts:
Zigbee2MQTT-Version
    [1.39.0]

Coordinator-Typ
    zStack3x0

Coordinator-Version
    20230507

Coordinator IEEE Adresse
    0x00124b002a2f22a7

Frontend-Version
    0.7.2

Zigbee Herdsman Konverter Version
    19.62.0

Zigbee Herdsman Version
    0.50.1

**Reply by francisp2** [reply]
see https://github.com/Koenkk/zigbee2mqtt/issues/23238#issuecomment-2204103062
